### DISCUSSION TITLE
ios actions widget

### DISCUSSION THREAD

**Discussion by nicolasparada** [community/community#35107] [Mobile]
I always think that would be nice to have an iOS widget to see running actions :)

**Comment by HenSquared** [comment]
Hi @nicolasparada,

Thanks for the feature request! I like this idea.
 We just shipped support for [actions](https://github.blog/changelog/2022-10-04-introducing-actions-on-github-mobile/) and released a new widget in the most recent release. So this is a fun combination of those two. :) 

I've added this feature request to our internal tracking issue. As we receive similar feature requests (and upvotes) we prioritize our work accordingly. 

Thanks a bunch for the idea!

**Comment by sami19693** [comment]
iPhone 13 Pro Max
